src/App.scss

```diff
@@ -98,9 +103,9 @@ $timelineCellLabelColor: yellow;
 		 display: grid;
 		 justify-content: center;
 		 align-items: center;
-
 		 > svg {
-			 width: 100%;
+            width: 100%;
+            color: $playButtonColor;
```

Comment:
seems unrelated

Reply:
Removed. I didn't think a single color change would be problematic when I already needed to add to the stylesheet for this task. I would've undone the changes if anyone disagreed in code review, but I can create a separate PR if need be.